### PR TITLE
Add Static Feature

### PR DESCRIPTION
**Pull request by KaQuMiQ**

### CODE REVIEW

**Comment by KaQuMiQ** [comment]
Depends on #13

**Review by mbalawajdermq — APPROVED** [review]

File: Sources/MQDo/Features/StaticFeature.swift [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+	/// Default instance of ``StaticFeature`` will be used
+	/// if not registered manually on ``RootScope`` features registry. This function does not have to return the same instance.
+	/// Treat it more like a factory method instead to avoid
+	/// unnecessary initialization before it is actually needed.
+	/// If ``defaultImplementation`` should not be available
+	/// leave it unimplemented (using default implementation).
+	///
+	/// Default implementation will not be used in test features
+	/// container. Test feature containers will always use
+	/// plceholder as a default implementation.
```
Comment (sq9rt-mq) [inline Sources/MQDo/Features/StaticFeature.swift:24]: typo

File: Sources/MQDo/Container/Features.swift [hunk 2]
```diff
@@ -175,20 +179,107 @@ extension Features {
...
 }
 
 extension Features {
 
-	/// Get an instance of the requested feature if able.
+	/// Access an instance of a ``StaticFeature``.
 	///
-	/// This function allows accessing instances of features.
-	/// Access to the feature depends on provided ``FeatureLoader``
+	/// This function allows to accessing instances of static features.
```
Comment (sq9rt-mq) [inline Sources/MQDo/Container/Features.swift:192]: Reword: ...allows for accessing instances of static features.

File: Sources/MQDo/Container/Features.swift [hunk 3]
```diff
@@ -175,20 +179,107 @@ extension Features {
...
+	/// Static features are always available and shared
+	/// across fatures container tree. All static features has
```
Comment (sq9rt-mq) [inline Sources/MQDo/Container/Features.swift:194]: ...features have to be...

**Review by sq9rt-mq — APPROVED** [review]

**Pull request merged**